style.css

```diff
@@ -117,13 +119,46 @@ ul li:hover ul {
 }
 /* Style for Navigation ends here*/
 
+.box1 {
+  box-sizing: border-box;
+  grid-column-start: 1;
+  grid-column-end: span 2;
+  padding-bottom: 10px;
+  height: 700px;
+  width: 1990px;
+  background-color: #3750A5;
+  color: #A2091B;
+}
+
+.box2 {
+  box-sizing: border-box;
+  grid-column-start: 1;
+  grid-column-end: span 2;
+  padding-bottom: 10px;
+  height: 20px;
+  width: 1990px;
+  background-color: #4DC9FF;
+  color: #A2091B;
+}
+
+.box3 {
+  box-sizing: border-box;
+  grid-column-start: 1;
+  grid-column-end: span 2;
+  padding-bottom: 10px;
+  height: 20px;
+  width: 1990px;
+  background-color: #87CEEB;
+  color: #FBCA01; 
+}
 
 /* style for slider starts here */
 .slider {
     position: relative;
-    width: 2000px;  /* adjustment to show each image on carousel. */
-	margin-right: 100px;
+    width: 1990px;  /* adjustment to show each image on carousel. */

```

Comment:
For the width of the slider I changeded it from 2000px to 1990px as it was 10px more then the width of the content.